testdata/10/code.go

```diff
@@ -0,0 +1,34 @@
+package code
+
+import (
+	"errors"
+	"net/http"
+	"regexp"
+)
+
+// myVar is just a bad named global var.
+var myVar = 1
+
+// ErrNotFound is an error and should be OK.
+var ErrNotFound = errors.New("this is error")
+
+// ErrIsNotErr is not an error and should not be OK.
```

Comment:
```suggestion
// ErrIsNotErr is an error and should be OK.
```

Missed this one reverting, this is still ok so the comment is wrong. The tests are passing though.